config/pathSetup.m

```diff
@@ -101,11 +63,48 @@
 %% Print out the paths
 disp(['------------------------------------------------'])
 disp(['Setting up the the following path parameters'])
-disp(['Local GitHub repository path = ' gitPath])
+disp(['Local GitHub repository path = ' autogenRootDir])
 disp(['Simulink model path          = ' mp.model_path])
 disp(['Test signals path            = ' mp.test_signals_path])
 disp(['VHDL Codegen path            = ' mp.ipcore_codegen_path])
 disp(['Driver Codegen path          = ' mp.driver_codegen_path])
 disp(['UI Codegen path              = ' mp.ui_codegen_path])
 disp(['Quartus path                 = ' mp.quartus_path])
-disp(['------------------------------------------------'])
\ No newline at end of file
+disp(['------------------------------------------------'])
+
+
+function config = getconfig()
+% Get the autogen configuration from path.json or automatically
+    config_dir = erase(mfilename('fullpath'), mfilename); 
+    config_path = config_dir + "path.json";
```

Comment:
Let's change these variable names to match our Matlab style guidelines, e.g. camelCase.

Reply:
Fixed